src/components/productInfo/ProductInfoPage.tsx

```diff
@@ -0,0 +1,53 @@
+import { useQuery } from "@tanstack/react-query";
+import { useParams } from "react-router-dom";
+import { getProductInformation } from "../../api/fakestoreApi";
+import { priceInRands } from "../../utils";
+import { useCartStore } from "../stores/useCartStore";
+
+const ProductInfoPage = () => {
+  const { productId } = useParams();
+
+  const query = useQuery({
+    queryKey: [productId],
+    queryFn: () => getProductInformation(productId),
+  });
+  const { adjustItemQuantity } = useCartStore();
+
+  if (!query.data) {
+    return;
+  }
+
+  return (
+    <div className="p-4 flex flex-col">
+      <div>
+        <div className="text-xl pb-2 font-semi-bold">{query.data?.title}</div>
```

Comment:
If you have your guard clause on line 16 do you need to do the optional chaining to check data?